analytics-integrations/taplytics/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply from: rootProject.file('gradle/integration.gradle')
+
+dependencies {
+  compile files('libs/Taplytics-1.4.3.jar')
```

Comment:
@VicV Do you have a version of the JAR that doesn't include all your dependencies? We have other libraries that depend on OkHttp for instance, and not using a package manager will cause conflicts.